packages/litexa/src/parser/litexa.pegjs

```diff
@@ -1166,6 +1182,18 @@ when "my name is $name"
   or "call me $name"
 ```
 
+2. The statement can be followed by [or](#or) intent name statements to have
+these intents share the same handler.
+
+```coffeescript
+when RephraseQuestionIntent
+  or AMAZON.RepeatIntent
+  or UnsureIntent
+```
+
+These types of subordinate statements cannot be mixed; doing so will result in a
+compilation-time error.
```

Comment:
Would use "compile time" when describing the time period during which compilation occurs. By contrast, "compilation time" is usually used to describe the duration.